### ISSUE TITLE
popup dialogs, menus are not modal to screen readers

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#605]
Although `aria-modal="true"` is used on the `dialog` itself (well, the `div` with the `dialog` `role`), this is not supported everywhere (for example, VoiceOver on MacOS continues to access the markup content beneath the modal layer). The `aria-hidden` attribute must not only be applied to the modal, but to the `inert` content beneath (visually), which is the main React root (structurally), separate from the app's React overlay `div` dedicated to modals.

https://github.com/readium/readium-desktop/blob/1263712b0ddab6ad43e3c95f45a9ed74c831960d/src/renderer/components/dialog/Dialog.tsx#L53-L54

See "Notes on aria-modal and aria-hidden" in https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html
See also: https://developer.paciellogroup.com/blog/2018/06/the-current-state-of-modal-dialog-accessibility/

Also, I am pretty sure `display: none` should be used instead of CSS `visibility: hidden`:
https://github.com/readium/readium-desktop/blob/1263712b0ddab6ad43e3c95f45a9ed74c831960d/src/renderer/components/dialog/Dialog.tsx#L57

**Comment by danielweck** [comment]
For popup menus:

https://github.com/readium/readium-desktop/blob/1263712b0ddab6ad43e3c95f45a9ed74c831960d/src/index_app.ejs#L8-L11

https://github.com/readium/readium-desktop/blob/1263712b0ddab6ad43e3c95f45a9ed74c831960d/src/renderer/components/utils/menu/MenuContent.tsx#L27-L61

**Comment by danielweck** [comment]
Fixed via https://github.com/readium/readium-desktop/commit/5a235e99460f162c887c90840473dbc8558f1272
